schema_dev.yml

```diff
@@ -1,10 +1,8 @@
 ruby:
-    - 2.3.1
+    - 2.4.1
```

Comment:
Really we should probably be testing on the oldest version of Ruby supported by rails rather than the newest version of Ruby (to make sure we're not depending on features of a newer version of ruby than others use).   That lowest allowable version according to rails's gemspec is 2.2.2; though Ruby 2.2.X is nearing EOL so maybe no point bothering with it now.

So probably we should test on 2.3.4 which is the latest in the 2.3.X series, or on both 2.3.4 and 2.4.1.   (Rails itself tests on 2.2.7, 2.3.4, and 2.4.1).

Anyway probably doesn't really matter much